js/about/newtab.js

```diff
@@ -43,21 +43,10 @@ class NewTabPage extends React.Component {
     const updatedStamp = sanitizedData.getIn(['newTabDetail', 'updatedStamp'])
 
     // Only update if the data has changed.
-    // console.log(updatedStamp + ']DATA_TS==LAST_TS[' + this.state.updatedStamp)
     if (updatedStamp === this.state.updatedStamp) {
       return
     }
 
-    // Ensure top sites only has unique values
```

Comment:
Since this logic is removed (which is OK), maybe we can move what's left into ipc.on() line :smile: I originally had added this because my state got into a situation where I had many duplicates, which caused problems